core/templates/pages/release-coordinator-page/features-tab/features-tab.component.ts

```diff
@@ -316,12 +336,12 @@ export class FeaturesTabComponent implements OnInit {
     return issues;
   }
 
-  get isDummyFeatureEnabled(): boolean {
-    return this.featureService.status.DummyFeature.isEnabled;
+  get isDummyFeatureFlagForE2ETestsEnabled(): boolean {
+    return this.featureService.status.DummyFeatureFlagForE2ETests.isEnabled;
   }
 
   async reloadDummyHandlerStatusAsync(): Promise<void> {
-    if (this.isDummyFeatureEnabled) {
+    if (this.isDummyFeatureFlagForE2ETestsEnabled) {
```

Comment:
Is this a function? If so then shouldn't it have a () after it?

If it's a variable then it should be dummyFeatureFlagForE2eTestsIsEnabled

Reply:
Yes, it is a function but the function is of type `get` so we would not require to have a (), this function is defined just above this line.
Thanks

Reply:
Yes, make sense to rename it. I have updated this.
Thanks